walkthroughs/howto-k8s-timeout-policy/deploy.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $AWS_ACCOUNT_ID ]; then
+    echo "AWS_ACCOUNT_ID environment variable is not set."
+    exit 1
+fi
+
+if [ -z $AWS_DEFAULT_REGION ]; then
+    echo "AWS_DEFAULT_REGION environment variable is not set."
+    exit 1
+fi
+
+if [ -z $ENVOY_IMAGE ]; then
+    echo "ENVOY_IMAGE environment variable is not set to App Mesh Envoy, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html"
+    exit 1
+fi
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+PROJECT_NAME="howto-k8s-timeout-policy"
+APP_NAMESPACE=${PROJECT_NAME}
+MESH_NAME=${PROJECT_NAME}
+CLOUDMAP_NAMESPACE="${PROJECT_NAME}.pvt.aws.local"
+
+ECR_IMAGE_PREFIX="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${PROJECT_NAME}"
+FRONT_APP_IMAGE="${ECR_IMAGE_PREFIX}/feapp"
+COLOR_APP_IMAGE="${ECR_IMAGE_PREFIX}/colorapp"
+
+MANIFEST_VERSION="${1:-v1beta2}"
+
+error() {
+    echo $1
+    exit 1
+}
+
+check_virtualnode_v1beta1(){
+    #check CRD
+    crd=$(kubectl get crd virtualnodes.appmesh.k8s.aws -o json | jq -r '.. | .cloudMap?.properties.namespaceName? | select(. != null)')
+    if [ -z "$crd" ]; then
+        error "$PROJECT_NAME requires virtualnodes.appmesh.k8s.aws CRD to support Cloud Map service-discovery. See https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md#v012"
+    else
+        echo "CRD check passed!"
+    fi
+}
+
+check_virtualnode_v1beta2(){
+    #check CRD
+    crd=$(kubectl get crd virtualnodes.appmesh.k8s.aws -o json | jq -r '.. | .awsCloudMap?.properties.namespaceName? | select(. != null)')
+    if [ -z "$crd" ]; then
+        error "$PROJECT_NAME requires virtualnodes.appmesh.k8s.aws CRD to support Cloud Map service-discovery. See https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md"
+    else
+        echo "CRD check passed!"
+    fi
+}
+
+check_appmesh_k8s() {
+    #check aws-app-mesh-controller version
+    if [ "$MANIFEST_VERSION" = "v1beta2" ]; then
+        currentver=$(kubectl get deployment -n appmesh-system appmesh-controller-manager -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'|tail -n1)
```

Comment:
`appmesh-controller-manager` -> `appmesh-manager` as per the helm charts [1]

[1] https://github.com/aws/eks-charts/tree/master/stable/appmesh-manager